torch/csrc/utils/tensor_layouts.cpp

```diff
@@ -21,6 +21,7 @@ void initializeLayouts() {
   // for now, let's look these up by Backend; we could create our own enum in the future.
   registerLayoutObject((THPLayout*)strided_layout, at::Backend::CPU);
   registerLayoutObject((THPLayout*)strided_layout, at::Backend::CUDA);
+  registerLayoutObject((THPLayout*)strided_layout, at::Backend::FPGA);
```

Comment:
just noting I think we should be able get rid of this so you don't have to change this (obviously it's ideal to be able to minimize your changes).  I'll look into this separately.